packages/create-plugin/src/commands/generate/prettify-files.ts

```diff
@@ -4,13 +4,13 @@ import fs from 'node:fs';
 
 const exec = promisify(nodeExec);
 
-export async function prettifyFiles(exportPath: string) {
+export async function prettifyFiles(exportPath: string, packageManagerExecute = 'npx') {
```

Comment:
instead of accepting the packageManagerExecute, can you use the new `getPackageManagerExecute` here?